config/locales/spotlight.en.yml

```diff
@@ -72,7 +72,7 @@ en:
       role:
         updated: "User has been updated."
         destroyed: "User has been removed."
-        batch_error: "There was a problem saving the users."
+        batch_error: "There was a problem saving the user(s)."
```

Comment:
Can we use i18n's pluralization support to handle both these cases a little cleaner?